docs/csharp/language-reference/attributes/nullable-analysis.md

```diff
@@ -139,6 +139,9 @@ That informs the compiler that any code where the return value is `false` doesn'
 
 :::code language="csharp" source="snippets/NullableAttributes.cs" ID="NullCheckExample" :::
 
+> [!NOTE]
+> The preceding example is only valid in C# 11 and later. Starting with C# 11, the `nameof` expression can reference parameter and type parameter names when used in an attribute applied to a method. Prior to C# 11, you'll need to use a string literal instead of the `nameof` expression.
```

Comment:
```suggestion
> The preceding example is only valid in C# 11 and later. Starting with C# 11, the `[nameof` expression](../operators/nameof.md) can reference parameter and type parameter names when used in an attribute applied to a method. In C# 10 and earlier, you need to use a string literal instead of the `nameof` expression.
```